screening/openg2p.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "OpenG2P",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://docs.openg2p.org/"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://docs.openg2p.org/"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "nonPIIAccessMechanism": "Yes"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "OpenG2P is a tool used by adopters, the OpenG2P foundation by itself does not store any PII"
+    ],
+    "isLocIntlAdherent": "Yes",
+    "locIntlComplianceList": [
+      "OpenG2P is a tool used by adopters, the OpenG2P foundation by itself does not store any PII"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "w3c verifiable credentials and others under explorations especially around handling of PII "
+    ],
+    "evidenceStandardSupport": [
+      "Work in progress especially around compliance to open standards around handling and security of PII"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Principals for Digital Development; the Responsible Digital Payments Guidelines by the Better Than Cash Alliance"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "",
+      "additionalMitigationSteps": [
+        "https://openg2p.org/"
```

Comment:
A generic address like this one is not acceptable here. They should provide a more specific URL to provide the supporting evidence to the issue at hand.

Reply:
Replaced generic address with more detailed contextual response to the question as shown in [commit](https://github.com/unicef/publicgoods-candidates/pull/112/commits/b229d4a04fc4c6e9715bc226de432cfdd7ba2bca).